lbplanner/services/user/get_user.php

```diff
@@ -37,7 +37,7 @@ public static function get_user_parameters(): external_function_parameters {
         return new external_function_parameters(array(
             'userid' => new external_value(
                 PARAM_INT,
-                'The id of the user to get the data for. If not given userid is taken by Token',
+                'The Moodle id of the user to get the data for. If not given userid is taken by Token',
```

Comment:
again:

do we have our own id's for users? if not, annotating the ID as 'Moodle id' is unnecessary and could cause confusion